next/pages/api/user/verification/generate.ts

```diff
@@ -0,0 +1,21 @@
+import { withSentry } from '@sentry/nextjs'
```

Comment:
when we change this to use just a single example endpoint you can likely get rid of this generate one completely